fe1-web/src/features/evoting/components/__tests__/EventElection.test.tsx

```diff
@@ -0,0 +1,236 @@
+  messageId: '',
+  sender: '',
+  votes: [{ id: '', question: '', vote: [0] }],
+};
+
+const questionResult: QuestionResult = {
+  id: '',
+  result: [
+    { ballotOption: mockBallotOptions[0], count: 10 },
+    { ballotOption: mockBallotOptions[1], count: 3 },
+  ],
+};
+
+const notStartedElection = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: 'An election',
+  version: STRINGS.election_version_identifier,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: [question],
+  electionStatus: ElectionStatus.NOT_STARTED,
+  registeredVotes: [],
+});
+
+const runningElection = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: 'An election',
+  version: STRINGS.election_version_identifier,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: [question],
+  electionStatus: ElectionStatus.RUNNING,
+  registeredVotes: [registeredVote],
+});
+
+const terminatedElection = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: 'An election',
+  version: STRINGS.election_version_identifier,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: [question],
+  electionStatus: ElectionStatus.TERMINATED,
+  registeredVotes: [registeredVote],
+});
+
+const finishedElection = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: 'An election',
+  version: STRINGS.election_version_identifier,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: [question],
+  electionStatus: ElectionStatus.FINISHED,
+  registeredVotes: [registeredVote],
+});
+
+const resultElection = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: 'An election',
+  version: STRINGS.election_version_identifier,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: [question],
+  electionStatus: ElectionStatus.RESULT,
+  registeredVotes: [registeredVote],
+  questionResult: [questionResult],
+});
+
+const undefinedElection = new Election({
+  lao: mockLaoIdHash,
+  id: mockElectionId,
+  name: 'An election',
+  version: STRINGS.election_version_identifier,
+  createdAt: TIMESTAMP,
+  start: TIMESTAMP,
+  end: CLOSE_TIMESTAMP,
+  questions: [question],
+  // @ts-ignore: Sonarcloud wants high test coverage, this is to hit the default case
+  electionStatus: 'undefined',
+  registeredVotes: [registeredVote],
+  questionResult: [questionResult],
+});
+
+beforeEach(() => {});
+
+describe('EventElection', () => {
+  describe('Not started election', () => {
+    it('renders correctly for an organizer', () => {
+      const component = render(
+        <EventElection election={notStartedElection} isOrganizer />,
+      ).toJSON();
+      expect(component).toMatchSnapshot();
+    });
+    it('renders correctly for an attendee', () => {
+      const component = render(
+        <EventElection election={notStartedElection} isOrganizer={false} />,
```

Comment:
Wouldn't the property then be defined as `undefined` (which evaluates to falsy) ?

Reply:
It is actually a rule from the Airbnb style convention, as you can find [there](https://airbnb.io/javascript/react/). Normally, the IDE would complain about it by the way.

![image](https://user-images.githubusercontent.com/33868847/156269814-7533dd3a-2fca-4318-a3f0-4267b3997e1b.png)

Reply:
It might be the case, since `EventElection` doesn't have default property value for `isOrganizer`. It was how we handled false booleans in props until now though, so it is probably done this way in the application (I actually have never seen a boolean being set to false in React props). I guess it's not an important issue anyway.